python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -244,12 +244,31 @@ def _to_int(value):
                                 multiple=True)
 
 
-skip_existing_option = MWOptionDecorator("--skip-existing",
-                                         help=unwrap("""If all Quantum outputs already exist in the output RUN
-                                                     collection then that Quantum will be excluded from the
-                                                     QuantumGraph. Requires the 'run` command's `--extend-run`
-                                                     flag to be set."""),
-                                         is_flag=True)
+skip_existing_in_option = MWOptionDecorator(
+    "--skip-existing-in",
+    callback=split_commas,
+    default=None,
+    metavar="COLLECTION",
+    multiple=True,
+    help=unwrap(
+        """If all Quantum outputs already exist in the the specified list of
+        collections then that Quantum will be excluded from the QuantumGraph.
+        """
+    )
+)
+
+
+skip_existing_option = MWOptionDecorator(
+    "--skip-existing",
+    is_flag=True,
+    help=unwrap(
+        """This option is equivalent to --skip-existing-in with the name of
+        the output RUN collection. Requires the 'run` command's `--extend-run`
```

Comment:
I dropped "Requires 'run\` ..." because it's not required anymore (but --extend-run implies --skip-existing. There was another instance of "'run\`", I changed that single quotes. In most other cases option names are used without backticks so I dropped those here too.